src/python/pants/jvm/resources.py

```diff
@@ -91,43 +96,45 @@ async def assemble_resources_jar(
 
     input_filenames = " ".join(shlex.quote(file) for file in sorted(input_files))
 
-    resources_jar_result = await Get(
-        ProcessResult,
-        Process(
-            argv=[
-                bash.path,
-                "-c",
-                " ".join(
-                    [
-                        "TZ=UTC",
-                        touch.path,
-                        "-t 198001010000.00",
-                        input_filenames,
-                        "&&",
-                        "TZ=UTC",
-                        zip.path,
-                        "-oX",
-                        output_filename,
-                        input_filenames,
-                    ]
-                ),
-            ],
-            description=f"Build resources JAR for {request.component}",
-            input_digest=resources_jar_input_digest,
-            output_files=output_files,
-            level=LogLevel.DEBUG,
-        ),
+    resources_jar_result = await execute_process_or_raise(
+        **implicitly(
+            Process(
+                argv=[
+                    bash.path,
+                    "-c",
+                    " ".join(
+                        [
+                            "TZ=UTC",
+                            touch.path,
+                            "-t 198001010000.00",
```

Comment:
What is this?

Reply:
Reset time on resource files to a single time so that the produced jar (i.e., a zip) is reproducible.

Reply:
Cool, that's what I assumed - but wasn't sure